smsVoIncident.js

```diff
@@ -0,0 +1,40 @@
+const qs = require('qs');
+const got = require('got');
+const _ = require('lodash');
+
+
+exports.handler = function(context, event, callback) {
+    const { ROUTING_KEY, VICTOROPS_TWILIO_SERVICE_API_KEY } = context;
```

Comment:
You could do the same thing for `event` that you're doing here for `context` and then `event.Body`, etc becomes `Body`

Reply:
that is a good idea.